nntrainer/include/nntrainer_error.h

```diff
@@ -3,8 +3,8 @@
  *
  * This library is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Library General Public
- * License as published by the Free Software Foundation; either
- * version 2 of the License, or (at your option) any later version.
+ * License as published by the Free Software Foundation; 
+ * version 2.1 of the License.
  *
```

Comment:
This license statement is wrong. NNTrainer is not LGPL-2.1. It is apache-2.0-only. So, this must be 
```
// SPDX-License-Identifier: Apache-2.0-only 
/**
 * Copyright (C) 2020 Jijoong Moon <jijoong.moon@samsung.com>
 */
/**
 * @file nntrainer_error.h
 * @date 03 April 2020
 * @brief NNTrainer Error Codes
 * @see	https://github.com/nnstreamer/nntrainer
 * @author Jijoong Moon <jijoong.moon@samsung.com>
 * @bug No known bugs except for NYI items
 */
```